modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheConfigurationSplitterImpl.java

```diff
@@ -115,4 +139,13 @@ public CacheConfigurationSplitterImpl(Marshaller marshaller) {
             throw new IgniteException("Failed to serialize field [fieldName=" + fieldName + ", value=" + val + ']', e);
         }
     }
+
+    /**
+     * @param cfg Cache configuration.
+     * @return {@code true} If local node is affinity node for cache.
+     */
+    private boolean isAffinity(CacheConfiguration cfg) {
```

Comment:
Let's change to CacheConfiguration<?, ?> to prevent "unchecked" warning.

Reply:
I don't see any reason why I should add wildcard. Wildcards and bounds are useful when we deal with covariance or contravariance. This is not a case. Moreover, there is not any wildcard in this class, why I should add one? No reason at all, except highliting in one well-known IDE

Reply:
1. Because it's java compiler warning
2. Because most people use an IDE that will highlight this code.
3. Because we trying to cleanup Ignite code.

Reply:
Only reasonable variant is @SuppressWarning. CacheConfiguration is generic just for compliance with jcache api, nothing more. Adding everywhere warning suppression is a concern, but at least areasonable variant. Adding everywhere wildcards is like a cargo cult.